app/views/layouts/_header.html.haml

```diff
@@ -0,0 +1,51 @@
+= javascript_include_tag 'bar_slider'
+
+%header.navbar.navbar-fixed-top.navbar-inverse.header-background
+  .container
+    = link_to image_tag("foodielogo.png", alt: "Foodie logo", width: "100"), "/home", id: "logo" 
+    %nav
+      %ul.nav.navbar-nav.navbar-right
+        %li= link_to "HOME", "/home"
+        %li= link_to "ABOUT", "/about"
+        %li= link_to "SERVICE", "/service"
+        %li= link_to "BLOG", "/blog"
+        %li= link_to "CONTACTS", "/contact"
+    .center
+      %h1.center Ordering Food
+      .line-container
+        %span.line
+        %span.dot
+        %span.line
+      %h2
+        Food for the souls
+  #site-main
+    .bar
+      .subnav-bar
+        .subnav-menu
+          CATEGORY
+          %i.fa-solid.fa-angles-right.icon
+          .subnav-content
+            %p= link_to "Entrees", "/entree"
+            %p= link_to "Second Course", "/second-course"
```

Comment:
we discussed today at the meeting, but you don't need a new page for each category. You should be on the same page but simply submit a form that sets a param (e.g. `?category='second_course'`) and filter the data from the backend